package exception1;

public class ExcHan1 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
 try {
   System.out.println(10/0);
      }
 catch (Exception E)
 {
	 System.out.println("Number Divided by zero");
	 System.out.println(10/2);
 }
 
	}

}
